DBSCE/Overview.md

```diff
@@ -0,0 +1,259 @@
+
+### Attestation Service
+
+A service that is responsible for verifying that the binding key is issued by the expected device and providing the attestation of the key to the IdP. The attestation service can be owned by the IdP or a third party. DBSC(E) relies on the attestation service to validate the binding statement and ensure that the binding key and the device key belong to the same device. We have added details on the specifics of the binding artifacts generated during the device registration process, and the validation of the binding statement in the [DBSC(E) Key Generation](#key-generation-specifics) section.
+
+#### Key Generation Specifics
+
+This section defines the artifacts of binding and how they help establish proof of binding.
+
+There are three artifacts that are generated during/after the device registration process, which are used to prove the device binding. These are the [binding key](#binding-key), the [attestation key](#attestation-key), and the [binding statement](#binding-statement).
+
+##### **Binding Key**
+
+A _binding key_ is an asymmetric key pair that is used to bind an auth cookie. It is identified by a key ID and it is the responsibility of the browser to remember key ID mapping to _Local Key Helper_ and _RP_ and to use it for DBSC signatures and key management. As there could be multiple `devices` on a single physical device and or many device registration clients on a single device (mentioned [above](#device-registration-client)), the key mapping is expected to be managed by the Browser and the Local Key Helper.
+
+This **binding key** is the same as defined in the DBSC proposal [here](https://github.com/WICG/dbsc?tab=readme-ov-file#maintaining-a-session) and is expected to be cryptographically attested by the [attestation key](#attestation-key)which is created in a secure enclave (TPM or Keyguard) on the original device. However, please note that in the context of the DBSC proposal, the binding key validation is not guaranteed to be attack free, as it can be generated by malware, if the malware is present on the device. In the context DBSC(E), if the binding key is generated during the [device registration process in a clean room](#device-registration), it is protected from any malware trying to infilterate the session.
+
+##### **Attestation Key**
+
+An _attestation key_ is generated during the device registration process and has the following properties:
+
+    1. It signs only the private/other keys that reside in the same secure enclave as the attestation key.
+    2. It cannot sign any external payload, or if it signs, it cannot generate an output that can be interpreted as an attestation statement.
+
+Since the attestation key can be uploaded only once to the backend at the moment of device registration, in the clean room, and there is no need to change this key unless the device loses it (Could be due to key rotation or similar operations).
```

Comment:
This sentence seems incomplete, remove since?

Reply:
Yes, missed it. Corrected.